### DISCUSSION TITLE
19,521 days!!!!

### DISCUSSION THREAD

**Discussion by dantearmok** [transmission/transmission#5692] [Ideas]
Um, I think there's some wormhole math here, probably due to the run-until-crash cycle I've got going with the Nightly builds...

![19,521 days Screen Shot 2023-06-29 at 11 48 23 AM](https://github.com/transmission/transmission/assets/20881295/13675279-ccbc-4c0e-95b3-1db561b34f00)

**Comment by g-regor** [comment]
That means it started on 1st January 1970 ie. the unix timestamp is 0 ie. the download start time was not set.

And why is this in the Ideas category?
